crates/iota-e2e-tests/tests/dynamic_committee_tests.rs

```diff
@@ -122,16 +122,14 @@ impl StressTestRunner {
     }
 
     pub fn pick_random_active_validator(&mut self) -> IotaValidatorSummary {
```

Comment:
```suggestion
    pub fn pick_random_comittee_member(&mut self) -> IotaValidatorSummary {
```

Reply:
Thanks, addressed in [bc940d0](https://github.com/iotaledger/iota/pull/6015/commits/bc940d007168d3abebefc86ee33de3233c9bcb1b).